pkg/omf/functions/packages/omf.packages.hook.fish

```diff
@@ -0,0 +1,20 @@
+function omf.packages.hook -a path hook
```

Comment:
Not being nitpicky, but `omf.packages.hook.run` sounds more descriptive regarding to the action which the function performs.